### PR TITLE
Add virt-install, Ansible in requirements

### PR DESCRIPTION
**Pull request by trishnaguha**

### CODE REVIEW

File: requirements.yml [hunk 1]
```diff
@@ -5,8 +5,11 @@
   with_items:
   - git
   - rpm-ostree
+  - ansible
```
Comment (gbraad) [inline requirements.yml]: ansible can not install ansible from an ansible playbook

**Review by gbraad — CHANGES_REQUESTED** [review]
chicken/egg problem... needs to be solved with documentation?

File: requirements.yml (hunk 1 above)
Comment (trishnaguha) [reply requirements.yml]: Lol. stupid me!!

**Comment by trishnaguha** [comment]
@gbraad Modified :-).

**Review by gbraad — APPROVED** [review]

**Pull request merged**